src/login/LoginPage.jsx

```diff
@@ -167,6 +167,7 @@ class LoginPage extends React.Component {
     const isInstitutionAuthActive = !!secondaryProviders.length && !currentProvider;
     const isSocialAuthActive = !!providers.length && !currentProvider;
     const isEnterpriseLoginDisabled = getConfig().DISABLE_ENTERPRISE_LOGIN;
+    const ThirdPartyAuthPreloader = isSocialAuthActive || (isEnterpriseLoginDisabled && isInstitutionAuthActive);
 
     return (
       <>
```

Comment:
The condition at line 174 is the same as used in `ThirdPartyAuthPreloader`. Can we use it at line 174 also? Maybe we can give it a name that works for both conditions here?

Reply:
Changed to isThirdPartyAuthActive